citizennet/purescript-halogen-select
# Update `Select` for PureScript `0.12` and Halogen `4.0.0`

## What does this pull request do?

This PR updates Select for compatibility with Halogen 4.0.0 and PureScript 0.12.

## How should this be manually tested?

Run `yarn watch-docs`, navigate to the Examples page, and verify the console reports no errors and the typeahead and dropdown function as expected.

## Other Notes:

Project documentation will need an update and the library needs a major version bump. We'll probably need to do another major version bump as soon as Halogen moves to `5.0.0`.

Fixes #24
